ivy/functional/backends/paddle/statistical.py

```diff
@@ -167,7 +166,10 @@ def std(
     return _std(x, axis, correction, keepdims).cast(x.dtype)
 
 
-@with_unsupported_dtypes({"2.5.2 and below": ("int8", "uint8")}, backend_version)
+@with_supported_dtypes(
```

Comment:
Also the paddle backend gave this error so changed its decorator `ivy.utils.exceptions.IvyBackendException: paddle: execute_with_gradients: paddle: nested_map: paddle: mean: (NotFound) The kernel with key (CPU, Undefined(AnyLayout), int32) of kernel mean is not registered. Selected wrong DataType int32. Paddle support following DataTypes: float64, complex128, float32, complex64, bool`
I dont know why `mean` is getting triggered even when we are dealing with `sum` function.

Reply:
`mean` is actually applied for the gradient tests [here](https://github.com/unifyai/ivy/blob/9d6dcbd0dabce5371f1973823c5291ab7002bbdb/ivy_tests/test_ivy/helpers/function_testing.py#L1123), so both `mean` and `sum` are called when running the gradient tests for `sum`. We should probably we fixing `mean` first if there's an issue with `mean`. Thanks @Aaryan562 😄
Also, does paddle not natively support `int` dtypes for its `mean` and `sum` functions (cc @MahmoudAshraf97)?

Reply:
[`int8`, `int16`, `uint8`] are not supported for `mean` and [`int8`, `uint8`] for `sum`

Reply:
oh okay, ig then `int32` and `int64` can also be marked as supported @Aaryan562 😄
(probably specifying the unsupported dtypes is more convenient that specifying the supported dtypes for this function)

Reply:
`ivy.utils.exceptions.IvyBackendException: paddle: execute_with_gradients: paddle: nested_map: paddle: mean: (NotFound) The kernel with key (CPU, Undefined(AnyLayout), int32) of kernel mean is not registered. Selected wrong DataType int32. Paddle support following DataTypes: float64, complex128, float32, complex64, bool`

@vedpatwardhan But then when supporting `int32` and `int64` i am getting the above error

Reply:
I further debugged I got to know it was when my `test_gradients=True`. So I changes my test_sum to make it false and all the test cases passed succesfully. But what could be the reason for it?